torch/library.py

```diff
@@ -16,6 +18,7 @@
     'fallthrough_kernel',
     'impl_abstract',
     'get_ctx',
+    'custom_op',
```

Comment:
I'm surprised this is not tripping the public API checks. The `__module__` being private shouldn't be ok...

Reply:
It should definitely fail: https://github.com/pytorch/pytorch/blob/72662bf05b3499ce96aae9183a489c78f0c44c84/test/test_public_bindings.py#L477
Also because of this, we miss the fact that this is not properly documented on the website.

Reply:
Lol we need to update the public bindings test to catch these cases